src/main.ts

```diff
@@ -85,6 +89,7 @@ async function run() {
     await createAnnotations({ githubToken, newVersion: tagMsg, linesReplaced })
     core.setOutput('version', newVersion)
     core.setOutput('prefix', prefix)
+    core.setOutput('trailer', trailer)
```

Comment:
issue: don't forget to add an output to `action.yml`. I'm not clear why we need this as an output though.